src/06-sort.js

```diff
@@ -23,7 +27,18 @@ function sortByRuntimeAscending(songs) {}
  * @param {Object[]} songs - An array of songs. See the song data for more.
  * @returns {Object[]}
  */
-function sortByArtistNameDescending(songs) {}
+function sortByArtistNameDescending(songs) {
+  // using the .sort method to arrange the song objects in order by their artist name (descending)
+  return songs.sort((songA, songB) => {
+    // declaring variables to represent the value of the .artist keys in the song objects lowercased 
+    let keyA = songA.artist.toLowerCase();
+    let keyB = songB.artist.toLowerCase();
+
+    // using .localeCompare to return a number to indicate new position of reference string 
+    // comparing keyA to keyB to return song objects arranged in descending alphabetical order
+    return keyB.localeCompare(keyA)
```

Comment:
That's one way to do it! Another approach is using the attached logic to create conditionals that are more in line with how sort works
![Screenshot 2022-11-22 at 11 10 55 AM](https://user-images.githubusercontent.com/71154265/203365416-36134d88-7803-4859-ba4b-5d71c15e4e10.png)